jinja/node.go

```diff
@@ -0,0 +1,50 @@
+package jinja
+
+import "fmt"
+
+type NodeInter interface {
+	String() string
+}
+
+type Node struct {
```

Comment:
Node should probably be an interface, and remove NodeInter, and then specific node types can implement the Node interface.